js_modules/dagster-ui/packages/ui-core/src/instigation/TickDetailsDialog.tsx

```diff
@@ -133,7 +133,7 @@ const TickDetailsDialogImpl = ({tickId, tickResultType, instigationSelector}: In
           <>
             <span>Tick for {instigationSelector.name}: </span>
             <TimestampDisplay
-              timestamp={tick.timestamp}
+              timestamp={tick.scheduledExecutionTimestamp ?? tick.timestamp}
```

Comment:
This all looks good but I want to sanity check the ordering here, I'd assumed from the PR description that the old behavior was the inverse of this -- the time it actually executed, if it did execute, and the time it is scheduled for, if it did not yet execute.  If that's the case I think these would be `tick.timestamp ?? tick.scheduledExecutionTimestamp`, since `tick.scheduledExecutionTimestamp` is the one that is always present?

Reply:
edit: Ahh I see, maybe this is falling back to `tick.timestamp` for back-compat so this is more of a "new version ?? old version". Carry on!